### ISSUE TITLE
Add livereloadable react-router example

### ISSUE THREAD

**Issue by milankinen** [milankinen/livereactload#119]
Continuing from #43.

LiveReactLoads need an example for `react-router` (and perhaps also an example where both `react-router` and `redux` are being used). I'm gonna use @tukkajukka's excellent [example project](https://github.com/lemanse/simple-react-app) as a template (if it's ok?). 

The example will be a part of the upcoming `2.3.0` release (please see #118).

**Comment by tukkajukka** [comment]
@milankinen Yes, that is totally ok!

**Comment by neemzy** [comment]
Hi,

Any news regarding this example? :)

Thanks!

**Comment by tukkajukka** [comment]
@neemzy I just updated my React template and it uses now Redux, React Router and Livereactload. You can find that here: https://github.com/lemanse/simple-react-app

**Comment by neemzy** [comment]
@tukkajukka Thanks a ton, but my project isn't using Redux, and I therefore can't use `syncHistoryWithStore`. I'd really love to see a working example without Redux!

**Comment by tukkajukka** [comment]
There's a version without Redux if you checkout an earlier commit: https://github.com/lemanse/simple-react-app/commit/108f298cdac25334dd5c4fb3c5031d593f1720f2

**Comment by neemzy** [comment]
Oh, right. I'll give it a try soon then :) Cheers! Feel free to close the issue if you want to.

**Comment by milankinen** [comment]
I've had no time to work with this yet. Sorry about that. I'll try to finish this as soon as I manage to get some time for my personal projects.

**Comment by neemzy** [comment]
No worries! Thanks :)

**Comment by neemzy** [comment]
I don't think this is really required anymore as the lib works seamlessly with `react-router` :)

**Comment by milankinen** [comment]
Nice to hear that! I think it's still good to have an example though, thus not closing this issue yet. 😄
